### PR TITLE
Return team members grouped by type

### LINKED ISSUE #13: API: Adapt team api grouped by type

### PR DESCRIPTION
**Pull request by dricazenck**
Adjusted the team api to follow the UI proved: 

image
image
image

```
{
  "data": {
    "title": "Meet our team",
    "subtitle": "Core Team",
    "description": "The core team of our community is composed of visionary directors and dynamic leaders, who bring a wealth of experience and dedication to empowering women in technology.",
    "membersByType": {
      "Directors": [
        {
          "fullName": "Irina Kamalova",
          "position": "Senior Software Engineer",
          "image": {
            "path": "https://womencodingcommunity.com/assets/images/team/8.jpeg",
            "alt": "Director profile picture"
          },
          "network": [
            {
              "type": "Linkedin",
              "link": "https://www.linkedin.com/in/irina-kamalova/"
            },
            {
              "type": "Medium",
              "link": "https://iren-kamalova.medium.com/"
            }
          ]
        }
      ],
      "Leads": [
        {
          "fullName": "Eleonora Belova",
          "position": "Test Automation Engineer",
          "image": {
            "path": "https://womencodingcommunity.com/assets/images/team/7.jpeg",
            "alt": "Leader profile picture"
          },
          "network": [
            {
              "type": "Linkedin",
              "link": "https://www.linkedin.com/in/eleonora-belova-7069baa5/"
            },
            {
              "type": "Github",
              "link": "https://github.com/nora-weisser"
            }
          ]
        }
      ],
      "Evangelists": []
    }
  },
  "metadata": {
    "pagination": {
      "totalItems": 50,
      "totalPages": 2,
      "currentPage": 1,
      "pageSize": 25
    }
  }
}
```

Closes #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,40 +81,45 @@ The output should be like this:
...
-          },
-          {
-            "type": "Github",
-            "link": "https://github.com/nora-weisser/"
-          }
-        ]
-      }
-    ]
+    "membersByType": {
+      "Director": [
```
Comment (stepsen89) [inline README.md:85]: I would call it "directors"

File: README.md [hunk 2]
```diff
@@ -81,40 +81,45 @@ The output should be like this:
...
+        {
+          "fullName": "Irina Kamalova",
+          "position": "Senior Software Engineer",
+          "memberType": "Director",
```
Comment (stepsen89) [inline README.md:89]: if they are all the same member type, we wouldn't need this on each member

File: README.md [hunk 3]
```diff
@@ -81,40 +81,45 @@ The output should be like this:
...
+          "imagePath": "https://womencodingcommunity.com/assets/images/team/8.jpeg",
```
Comment (stepsen89) [inline README.md:90]: Will it be okay to use "Profile picture of {name}" for the alt tag?

File: README.md [hunk 4]
```diff
@@ -81,40 +81,45 @@ The output should be like this:
...
+              "link": "https://www.linkedin.com/in/irina-kamalova/"
+            },
+            {
+              "type": "Medium",
+              "link": "https://iren-kamalova.medium.com/"
+            }
+          ]
+        }
+      ],
+      "Lead": [
```
Comment (stepsen89) [inline README.md:103]: same here: "leads" and the one below "evangelists"

File: README.md (hunk 3 above)
Comment (dricazenck) [reply README.md:90]: @lilyndk I guess this question is for you, but I think anyway I will add the alt option for all images.

**Review by stepsen89 — APPROVED** [review]

**Pull request merged**